ui/src/features/Canvas/index.tsx

```diff
@@ -54,6 +54,7 @@ type Props = {
   onCut?: (isCutByShortCut?: boolean, node?: Node) => void;
   onPaste?: () => void;
   onPaneMouseMove?: (event: MouseEvent<Element, globalThis.MouseEvent>) => void;
+  onSpotlightUserDeselect?: () => void;
```

Comment:
Make this a generic `handlePaneClick` so that its ready to go if we need to add additional checks/etc easily